lib/game_maker.ex

```diff
@@ -0,0 +1,156 @@
+defmodule GameMaker do
```

Comment:
This looks a lot better than before. Nice job. 

Can you think of a better name than "GameMaker"?